### ISSUE TITLE
`[autoCloseOnContentClick]="true"` won't work if there is a duplicate instance of `<hc-pop>`

### ISSUE THREAD

**Issue by pslyman** [HealthCatalyst/Fabric.Cashmere#1029]
**Summary**

`[autoCloseOnContentClick]="true"` won't work if there is a duplicate reference to the same `<hc-pop>`.
An example of this is having a two of `#helpMenu` in the navbar; one for the desktop mode, and one for the mobile collapsed menu (two total references). 

**Additional context**

In this screenshot, there is one `<hc-pop>`, but it's used in two places. `[autoCloseOnContentClick]="true"` won't work unless I remove one of the uses. 

![image](https://user-images.githubusercontent.com/22920382/66781360-49100a00-ee90-11e9-8177-debf8ad2fb1c.png)

In the expanded navbar:

`<hc-icon class="hc-navbar-icon" fontSet="fa" fontIcon="fa-question-circle-o" [hcPop]="helpMenu"></hc-icon>`

In the mobile collapsible menu:

```
<hc-navbar-mobile-menu>
...
   <hc-icon class="hc-navbar-icon mobile-options" fontSet="fa" fontIcon="fa-question-circle-o" [hcPop]="helpMenu"></hc-icon>
...
</hc-navbar-mobile-menu>
```

Will break `autoCloseOnContentClick` inside of this:
```
<hc-pop #helpMenu [autoCloseOnContentClick]="true">
...
</hc-pop>
```
Remove one of the instances, the flag starts working again.

**Comment by pslyman** [comment]
@corykon

**Comment by andrew-frueh** [comment]
Good catch @pslyman - I encountered this same bug when I was working on building right-click support into HcPop.  I submitted a fix that seems to resolve the issue in #1035 - let me know if you have any thoughts on the PR.

**Comment by pslyman** [comment]
PR seems good. Once it's merged, I will upgrade and test it out. Thanks for fixing this so quickly!
